sql/binding_contexts.cc

```diff
@@ -66,12 +66,7 @@ doGetFunction(const Utf8String & tableName,
     result.exec = [=] (const std::vector<ExpressionValue> & args,
                        const SqlRowScope & context)
         {
-            auto & row = static_cast<const RowContext &>(context);
-            //cerr << "rebinding to apply function " << functionName
-            //<< ": context type is "
-            //<< ML::type_name(context) << " outer type is "
-            //<< ML::type_name(row.outer) << endl;
-
+             const RowContext & row = dynamic_cast<const RowContext&>(context);
```

Comment:
The dynamic cast goes away right?  Also the commented cerr turned out to be useful for me a couple of time.  I would leave it.